.changes/next-release/bugfix-caf370b9-c943-4ece-a4c0-81bcefa7a4f5.json

```
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Q dev add error messages for upload URL expiry"
```

Please put more effort into these descriptions
```suggestion
  "description" : "Amazon Q Feature Dev: Add error messages when the upload URL expires"
```